src/Services/NodeBalanceService.ts

```diff
@@ -32,12 +32,13 @@ export class NodeBalanceService {
                     .div(earliestRecord).multipliedBy(100).toFixed(2) + '%'
             }
         } else {
-            return {
-                currentBalance: 0,
-                updatedAt: moment.utc(),
-                balanceChange: 0,
-                balanceChangePerc: 0 + '%'
-            }
+            return await NodeBalance.findOne({
```

Comment:
maybe this should be else if? and else part would stay as before if this is potentially null (no records).
Would be great if we could keep the same response structure in this one too (balance change and perc as 0).

Reply:
What should we return for updatedAt when there are no records? Now or 0

Reply:
Same as before is fine (now - moment.utc())